src/cls/rbd/cls_rbd.h

```diff
@@ -229,15 +229,15 @@ struct cls_rbd_snap {
     o.push_back(new cls_rbd_snap{});
     o.push_back(new cls_rbd_snap{1, "snap", 123456,
                                  RBD_PROTECTION_STATUS_PROTECTED,
-                                 {{1, "", "image", 123}, 234}, 31, {},
+                                 {}, 31, {},
```

Comment:
Nit: With no `parent` specification this would fit on the previous line:
```
                                 RBD_PROTECTION_STATUS_PROTECTED, {}, 31, {},
```

Reply:
sure. wanted to have visually minimal change. changed as suggested.